core-web/libs/portlets/edit-ema/portlet/src/lib/dot-ema-shell/store/dot-ema.store.ts

```diff
@@ -677,6 +675,19 @@ export class EditEmaStore extends ComponentStore<EditEmaState> {
         editorState
     }));
 
+    /**
+     * Update the editor state to scroll
+     *
+     * @memberof EditEmaStore
+     */
+    readonly setScrollingState = this.updater((state) => {
+        return {
+            ...state,
+            editorState: EDITOR_STATE.SCROLLING,
+            bounds: []
+        };
+    });
```

Comment:
`setScrollingState` vs `updateEditorScrollState` weird, not sure if is the naming.